app/models/setting/provisioning.rb

```diff
@@ -46,4 +55,10 @@ def validate_safemode_render(record)
       record.errors[:base] << _("Unable to disable safemode_render when bmc_credentials_accessible is disabled")
     end
   end
+
+  def self.default_global_templates
+    TemplateKind::PXE.map do |pxe_kind|
+      self.set("global_#{pxe_kind}", N_("Global default %s template that will not be affected by upgrades") % pxe_kind, nil, N_("Global default %s template") % pxe_kind)
```

Comment:
Can we show a select box instead of an input field? That might be easier for users to select an appropriate template.